scripts/testpop/statter.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/cactus/go-statsd-client/statsd"
+)
+
+// WriteCloserToSender is an adapter from io.WriteCloser to statsd.Sender
+type WriteCloserToSender struct {
+	bufWriter *bufio.Writer
+	closer    io.Closer
+}
+
+// Send implements the statsd.Sender interface
+func (w *WriteCloserToSender) Send(data []byte) (int, error) {
+	return w.bufWriter.Write(data)
+}
+
+// Close implements the statsd.Sender interface
+func (w *WriteCloserToSender) Close() error {
+	err := w.bufWriter.Flush()
+	w.closer.Close()
+	return err
+}
+
+// NewWriteCloserToSender returns a new WriteCloserToSender adapter
+func NewWriteCloserToSender(wc io.WriteCloser) statsd.Sender {
+	return &WriteCloserToSender{
+		bufWriter: bufio.NewWriter(wc),
+		closer:    wc}
+}
+
+// FormattedSender wraps a statsd.Sender and adds a timestamp and new lines
+// after each Send call.
+type FormattedSender struct {
+	s statsd.Sender
+}
+
+// Send implements the statsd.Sender interface
+func (fs *FormattedSender) Send(data []byte) (int, error) {
```

Comment:
I agree.
@severb could you chip in here and explain as the original author your preference?